docs/metrics-yaml-format.md

```diff
@@ -103,9 +106,12 @@ observation:
 
 Sematext App Agent supports the following metric data types:
 
-* `gauge`, `long_gauge`: Gauge data type. Default aggregation is `AVG`
-* `counter`, `long_counter`: Counter data type. Default aggregation is `SUM` 
-* `text`: Textual data type. Typically used for metrics that have to be extracted as tags
+* `gauge`, `long_gauge`:  Gauge is a metric that represents a single numerical value that can arbitrarily go up and down.
```

Comment:
After "... up and down. " please add - " Agent reports gauge metrics as their current value".